src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -25,6 +25,13 @@ namespace Clusters {
 namespace detail {
 // Structs shared across multiple clusters.
 namespace Structs {
+
+{{#zcl_enums}}
+{{#if has_more_than_one_cluster}}
+using {{asUpperCamelCase name}} = Clusters::detail::{{asUpperCamelCase name}};
+{{/if}}
+{{/zcl_enums}}
```

Comment:
Again: why is this bit needed?  This is going to create problems any time an enum and a struct have the same name, isn't it, by importing these names into the Structs namespace.

But most importantly: this should not be needed.  Just using the bareword name for the enum should find it now that it's in `Clusters::detail`.

I just tried taking this piece out and regenerating, and everything compiles fine.  Please take this out.

Reply:
It was needed when the enums were in detail::Enum, but now that they're just in detail I'll remove.